bay/images/awx-ee/context/_build/bindep.txt

```diff
@@ -0,0 +1,23 @@
+php ==7.4
+php-cli
+php-json
+php-pdo
+php-dom
+php-gd
+php-zip
+php-mbstring
+php-phar
+
+which
+git
+git-lfs
+patch
+
+unzip
+zip
+
+ca-certificates
+curl
+openssl
+jq
```

Comment:
So `gojq` not supported in bindep.txt?

Reply:
No, I logged into the running ee pod and ran a dnf search to check. Nothing in the Gojq docs either. It _might_ be in another repository but then we'd be messing about with adding that to the env and running the risk of unsupported packages if there were a breach.